src/Castle.Facilities.AspNetCore.Tests/Castle.Facilities.AspNetCore.Tests.csproj

```diff
@@ -20,4 +20,23 @@
 	  <ProjectReference Include="..\Castle.Facilities.AspNetCore\Castle.Facilities.AspNetCore.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <None Update="Fakes\ModelFakes.tt">
+	    <Generator>TextTemplatingFileGenerator</Generator>
```

Comment:
@jonorossi hope you are OK with this, otherwise I will convert this to code files.

Reply:
I wish I could replace some of the fluff in Windsor with this. Man there is a whole heap of test boiler plate hanging around.